custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_person.yaml

```
@@ -21,14 +21,14 @@ card_person:
         return states[variables.ulm_address].state
       } else {
         let state = states[variables.ulm_card_person_entity].state;
-        return variables["ulm_person_state_" + state] ? variables["ulm_person_state_" + state] : state;
+        return (state == "home" || state == "not_home") ? hass.resources[hass["language"]]["component.person" + ".state._." + state] : state == "unavailable" ? variables.ulm_translation_unavailable : state;
```

Changed logic to also give option for `unknown`
```suggestion
        let state = states[variables.ulm_card_person_entity].state;
        let option = ["home", "not_home", "unavailable", "unknown"]
        return (option.includes(state)) ? variables.ulm_translation_state : state
```